### PR TITLE
Add Package chewbbaca-1.0

### PR DESCRIPTION
**Pull request by dfornika**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/chewbbaca/build.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+DEST="$PREFIX/bin/chewBBACA"
```
Comment (bgruening) [inline recipes/chewbbaca/build.sh]: Yould it make sense to use PREFIX/lib/ here?

File: recipes/chewbbaca/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  skip: True  # [osx]
+  number: 0
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - numpy 1.13.1
```
Comment (bgruening) [inline recipes/chewbbaca/meta.yaml]: Is this strict pinning here needed?
Also for biopython and prodigal ...

File: recipes/chewbbaca/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    - scipy >=0.13.3
+    - plotly 1.12.*
+    - biopython 1.70
+    - prodigal 2.6.3
+    - blast >=2.5.0
+
+test:
```
Comment (bgruening) [inline recipes/chewbbaca/meta.yaml]: Please add a test.

File: recipes/chewbbaca/meta.yaml (hunk 2 above)
Comment (dfornika) [reply recipes/chewbbaca/meta.yaml]: I've done my best to match the versions listed on the [chewBBACA repo](https://github.com/B-UMMI/chewBBACA) and in the `requirements.txt` file

File: recipes/chewbbaca/build.sh (hunk 1 above)
Comment (dfornika) [reply recipes/chewbbaca/build.sh]: Sure, I've made the change.

**Comment by bgruening** [comment]
Thanks @dfornika!

**Pull request merged**